pil_analyzer/src/pil_analyzer.rs

```diff
@@ -50,38 +59,47 @@ struct PILAnalyzer<T> {
     /// appear in the source.
     source_order: Vec<StatementIdentifier>,
     symbol_counters: Option<Counters>,
-    current_file: PathBuf,
-    current_line_starts: Vec<usize>,
 }
 
-struct ParsedFile<T> {
-    line_starts: Vec<usize>,
-    ast: PILFile<T>,
-}
-
-impl<T: FieldElement> ParsedFile<T> {
-    #[allow(clippy::print_stderr)]
-    pub fn parse(path: &Path, contents: &str) -> Self {
-        let line_starts = parser_util::lines::compute_line_starts(contents);
-        let ast = parser::parse(Some(path.to_str().unwrap()), contents).unwrap_or_else(|err| {
-            eprintln!("Error parsing .pil file:");
-            err.output_to_stderr();
-            panic!();
-        });
-        ParsedFile { line_starts, ast }
+// when starting from an already parsed PILFile there's no reference to source code.
+// Make it so each statement is considred to be in its own line.
+fn pilfile_set_one_statement_per_line<T>(pil: &mut PILFile<T>) {
```

Comment:
Can this be deleted now?

Reply:
the idea here was to have lines matching the .pil file generated in the previous pipeline step... but even if this would be useful, the analyzer is now the wrong place to do it. Will remove.